### PR TITLE
HZN-151: Documentation for Page Sequence Monitor

### PR DESCRIPTION
**Pull request by indigo423**
Add documentation to allow an administrator to create a Page Sequence Monitor.
Explain all available configuration parameters for the monitor and give useful examples.
Migration of examples from the PSM wiki page to the official documentation.

* JIRA: http://issues.opennms.org/browse/HZN-151
* Bamboo:

### CODE REVIEW

File: opennms-doc/guide-admin/src/asciidoc/text/service-assurance/monitors/PageSequenceMonitor.adoc [hunk 1]
```diff
@@ -0,0 +1,306 @@
...
+The overall layout of the monitor configuration is more complex.
+Additionally, it is possible to configure a page sequence containing a path through a web application.
+
+.Monitor parameters for the PageSequenceMonitor
+[options="header, autowidth"]
+|===
+| Parameter        | Description                                                                           | Required | Default value
+| `timeout`        | Time in milliseconds to wait before assuming that a packet has not responded          | optional | `3000`
+| `retry`          | The number of retries to attempt when a packet fails to respond in the given timeout  | optional | `0`
+| `strict-timeout` | ??                                                                                    | optional | `false`
```
Comment (indigo423) [inline opennms-doc/guide-admin/src/asciidoc/text/service-assurance/monitors/PageSequenceMonitor.adoc]: This parameter was found in the code but seems to have no function or is unclear what it does?

File: opennms-doc/guide-admin/src/asciidoc/text/service-assurance/monitors/PageSequenceMonitor.adoc [hunk 2]
```diff
@@ -0,0 +1,306 @@
...
+| `page-sequence`  | Definition of the page-sequence to execute, see table with _Page Sequence Parameter_  | required | `-`
+| `sequence-retry` | ??                                                                                    | ??       | ??
```
Comment (indigo423) [inline opennms-doc/guide-admin/src/asciidoc/text/service-assurance/monitors/PageSequenceMonitor.adoc]: This parameter was found in the code but seems to have no function or is unclear what it does? Does it has a default value?

File: opennms-doc/guide-admin/src/asciidoc/text/service-assurance/monitors/PageSequenceMonitor.adoc [hunk 3]
```diff
@@ -0,0 +1,306 @@
...
+|===
+
+.Page Sequence Parameter
+[options="header, autowidth"]
+|===
+| Parameter                  | Description                                                                                | Required | Default
+| `name`                     | The name of the page-sequence. *(Is this relevant/used?)*                                  | optional | `-`
```
Comment (indigo423) [inline opennms-doc/guide-admin/src/asciidoc/text/service-assurance/monitors/PageSequenceMonitor.adoc]: What is the function of the "name" of a page-sequence? Where is it used what is the function? Is there a default?

**Comment by mvrueden** [comment]
The `strict-timeout` defines if the monitor should wait until a failure or just retry.
If there is no retry set, the `strict-timeout` is not used.

If there is a `retry >= 1` and `strict-timeout` is `true` the poller waits `NOW - InitialAttempt ms + Timeout ms` (of course only if the result is positive).
With `strict-timeout = false` the next attempt is started right after a failure.

The parameter name `strict-timeout` may be a bit misleading.

**Pull request merged**